js/ui/pivot_grid.d.ts

```diff
@@ -303,16 +304,48 @@ export interface dxPivotGridOptions extends WidgetOptions<dxPivotGrid> {
       /**
        * @docid
        * @default false
+       * @deprecated
        */
       allowSearch?: boolean;
+      /**
+       * @docid
+       * @default true
+       */
+      allowSelectAll?: boolean;
       /**
        * @docid
        * @default 325
        */
       height?: number;
+      /**
+       * @docid
+       */
+      search?: {
+        /**
+         * @docid
+         * @default false
+         */
+        enabled?: boolean;
+        /**
+         * @docid
+         * @default {}
+         */
+        editorOptions?: any;
+        /**
+         * @docid
+         * @default 'contains'
+         */
+        mode?: SearchMode;
+        /**
+         * @docid
+         * @default 500
+         */
+        timeout?: number;
+      };
```

Comment:
Maybe better to just reuse interface from grids.d.ts

Reply:
Agree